recurly/__init__.py

```diff
@@ -18,8 +19,8 @@
 
 """
 
-
 __version__ = '2.2.17'
+__python_version__ = '.'.join(map(str, sys.version_info[:3]))
```

Comment:
I think this should provide enough clarity. If you'd like we could expand `version_info` to `[:5]` to additionally view the `releaselevel` and `serial`. Probably unnecessary though.